pkg/apis/admissionregistration/types.go

```diff
@@ -458,6 +469,14 @@ type ValidatingAdmissionPolicyBindingSpec struct {
 	//
 	// Required.
 	ValidationActions []ValidationAction
+
+	NamespaceParamRef *NameSpaceParamRef
+}
+
+type NameSpaceParamRef struct {
+	FailAction *ParamFailureActionType
+	Name       string
+	Selector   *metav1.LabelSelector
```

Comment:
+API docs on these fields

Some critical things to explain:

- Must set either Name or Selector but not both
- How FailAction works for both cases
- What happens with a Selector matches 0 param objects and when it matches more than one param objects

Reply:
added API docs